diff -Nur -x '*.orig' -x '*.rej' e-smith-packetfilter-1.18.0/root/etc/e-smith/templates/etc/ulogd.conf/basic mezzanine_patched_e-smith-packetfilter-1.18.0/root/etc/e-smith/templates/etc/ulogd.conf/basic
--- e-smith-packetfilter-1.18.0/root/etc/e-smith/templates/etc/ulogd.conf/basic	2007-05-18 22:01:23.000000000 -0600
+++ mezzanine_patched_e-smith-packetfilter-1.18.0/root/etc/e-smith/templates/etc/ulogd.conf/basic	2007-05-18 21:25:38.000000000 -0600
@@ -4,8 +4,8 @@
 loglevel=5
 rmem=131071
 bufsize=150000
-plugin="/usr/lib/ulogd/ulogd_BASE.so"
-plugin="/usr/lib/ulogd/ulogd_LOGEMU.so"
+plugin={ -f "/usr/lib64/ulogd/ulogd_BASE.so" ? "/usr/lib64/ulogd/ulogd_BASE.so" : "/usr/lib/ulogd/ulogd_BASE.so" }
+plugin={ -f "/usr/lib64/ulogd/ulogd_LOGEMU.so" ? "/usr/lib64/ulogd/ulogd_LOGEMU.so" : "/usr/lib/ulogd/ulogd_LOGEMU.so" }
 
 [LOGEMU]
 file="/dev/stdout"
