diff -Nur -x '*.orig' -x '*.rej' e-smith-packetfilter-1.16.0/root/etc/e-smith/templates/etc/rc.d/init.d/masq/90InboundTCP10filter_tcp mezzanine_patched_e-smith-packetfilter-1.16.0/root/etc/e-smith/templates/etc/rc.d/init.d/masq/90InboundTCP10filter_tcp
--- e-smith-packetfilter-1.16.0/root/etc/e-smith/templates/etc/rc.d/init.d/masq/90InboundTCP10filter_tcp	2006-03-16 13:38:35.000000000 +1100
+++ mezzanine_patched_e-smith-packetfilter-1.16.0/root/etc/e-smith/templates/etc/rc.d/init.d/masq/90InboundTCP10filter_tcp	2006-07-26 12:02:36.950612845 +1000
@@ -37,11 +37,5 @@
 	--destination \$OUTERNET --src $host --jump ACCEPT
 HERE
 	}
-
-	$OUT .= <<HERE;
-    /sbin/iptables -A \$NEW_InboundTCP --proto tcp --dport $port \\
-	--destination \$OUTERNET --jump denylog
-
-HERE
     }
 }
diff -Nur -x '*.orig' -x '*.rej' e-smith-packetfilter-1.16.0/root/etc/e-smith/templates/etc/rc.d/init.d/masq/90InboundUDP10filter_udp mezzanine_patched_e-smith-packetfilter-1.16.0/root/etc/e-smith/templates/etc/rc.d/init.d/masq/90InboundUDP10filter_udp
--- e-smith-packetfilter-1.16.0/root/etc/e-smith/templates/etc/rc.d/init.d/masq/90InboundUDP10filter_udp	2006-03-16 13:38:35.000000000 +1100
+++ mezzanine_patched_e-smith-packetfilter-1.16.0/root/etc/e-smith/templates/etc/rc.d/init.d/masq/90InboundUDP10filter_udp	2006-07-26 12:01:59.435094267 +1000
@@ -37,11 +37,5 @@
 	--destination \$OUTERNET --src $host --jump ACCEPT
 HERE
 	}
-
-	$OUT .= <<HERE;
-    /sbin/iptables -A \$NEW_InboundUDP --proto udp --dport $port \\
-	--destination \$OUTERNET --jump denylog
-
-HERE
     }
 }
