diff -Nur -x '*.orig' -x '*.rej' e-smith-packetfilter-1.17.0/root/etc/e-smith/templates/etc/rc.d/init.d/masq/00Definitions mezzanine_patched_e-smith-packetfilter-1.17.0/root/etc/e-smith/templates/etc/rc.d/init.d/masq/00Definitions
--- e-smith-packetfilter-1.17.0/root/etc/e-smith/templates/etc/rc.d/init.d/masq/00Definitions	2006-08-13 06:33:35.000000000 -0400
+++ mezzanine_patched_e-smith-packetfilter-1.17.0/root/etc/e-smith/templates/etc/rc.d/init.d/masq/00Definitions	2006-08-28 03:54:50.000000000 -0400
@@ -7,6 +7,10 @@
 
     INTERNALIF=$internalif
     OUTERNET=$outernet
+    if [ -z "\$OUTERNET" ]
+    then
+        OUTERNET=1.1.1.1 # Put in placeholder address, to ensure correct iptables syntax
+    fi
 HERE
 
     if (defined $ExternalInterface{Name})
