diff -Nur -x '*.orig' -x '*.rej' e-smith-packetfilter-1.15.1/root/etc/e-smith/templates/etc/rc.d/init.d/masq/90InboundTCP10filter_tcp mezzanine_patched_e-smith-packetfilter-1.15.1/root/etc/e-smith/templates/etc/rc.d/init.d/masq/90InboundTCP10filter_tcp
--- e-smith-packetfilter-1.15.1/root/etc/e-smith/templates/etc/rc.d/init.d/masq/90InboundTCP10filter_tcp	2003-09-26 00:46:00.000000000 +1000
+++ mezzanine_patched_e-smith-packetfilter-1.15.1/root/etc/e-smith/templates/etc/rc.d/init.d/masq/90InboundTCP10filter_tcp	2005-07-20 17:50:17.000000000 +1000
@@ -6,6 +6,8 @@
 
         my $port = $props{TCPPort};
 
+	my $deny_hosts = $props{DenyHosts} || '';
+
 	my $allow_hosts = $props{AllowHosts} || '0.0.0.0/0';
 
 	unless ( ($props{status} || 'disabled') eq 'enabled')
@@ -18,7 +20,15 @@
 	    $allow_hosts = '';
 	}
 
-	$OUT .= "    # " . $filter->key . ": TCPPort $port, AllowHosts: $allow_hosts\n";
+	$OUT .= "    # " . $filter->key . ": TCPPort $port, AllowHosts: $allow_hosts, DenyHosts: $deny_hosts\n";
+
+	foreach my $host (split(',', $deny_hosts))
+	{
+	    $OUT .= <<HERE;
+    /sbin/iptables -A \$NEW_InboundTCP --proto tcp --dport $port \\
+	--destination \$OUTERNET --src $host --jump denylog
+HERE
+	}
 
 	foreach my $host (split(',', $allow_hosts))
 	{
