diff -Nur -x '*.orig' -x '*.rej' e-smith-packetfilter-1.18.0/root/etc/e-smith/templates.metadata/etc/ulogd.conf mezzanine_patched_e-smith-packetfilter-1.18.0/root/etc/e-smith/templates.metadata/etc/ulogd.conf
--- e-smith-packetfilter-1.18.0/root/etc/e-smith/templates.metadata/etc/ulogd.conf	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_e-smith-packetfilter-1.18.0/root/etc/e-smith/templates.metadata/etc/ulogd.conf	2007-04-06 09:34:48.000000000 -0600
@@ -0,0 +1 @@
+PERMS=0600
