diff -Nur -x '*.orig' -x '*.rej' e-smith-packetfilter-1.15.1/root/etc/e-smith/templates/etc/rc.d/init.d/masq/90InboundTCP05RejectIDENT mezzanine_patched_e-smith-packetfilter-1.15.1/root/etc/e-smith/templates/etc/rc.d/init.d/masq/90InboundTCP05RejectIDENT
--- e-smith-packetfilter-1.15.1/root/etc/e-smith/templates/etc/rc.d/init.d/masq/90InboundTCP05RejectIDENT	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_e-smith-packetfilter-1.15.1/root/etc/e-smith/templates/etc/rc.d/init.d/masq/90InboundTCP05RejectIDENT	2005-07-20 16:37:37.053795104 -0600
@@ -0,0 +1,6 @@
+
+    /sbin/iptables -A $NEW_InboundTCP --proto tcp --dport 113 \
+        --destination $OUTERNET \
+	--jump REJECT \
+	--reject-with tcp-reset
+
