diff -Nur -x '*.orig' -x '*.rej' e-smith-packetfilter-1.18.0/root/etc/e-smith/templates/etc/rc.d/init.d/masq/00Definitions mezzanine_patched_e-smith-packetfilter-1.18.0/root/etc/e-smith/templates/etc/rc.d/init.d/masq/00Definitions
--- e-smith-packetfilter-1.18.0/root/etc/e-smith/templates/etc/rc.d/init.d/masq/00Definitions	2007-01-26 18:51:31.000000000 -0700
+++ mezzanine_patched_e-smith-packetfilter-1.18.0/root/etc/e-smith/templates/etc/rc.d/init.d/masq/00Definitions	2007-04-09 03:14:17.000000000 -0600
@@ -13,7 +13,7 @@
     fi
 HERE
 
-    if (defined $ExternalInterface{Name})
+    if ($SystemMode ne "serveronly")
         { $OUT .= "    OUTERIF=".$ExternalInterface{Name} }
     else
         { $OUT .= "#    OUTERIF='there_isnt_one'"; }
