diff -Nur -x '*.orig' -x '*.rej' e-smith-packetfilter-1.16.0/root/etc/e-smith/templates/etc/rc.d/init.d/masq/10masq_ftp mezzanine_patched_e-smith-packetfilter-1.16.0/root/etc/e-smith/templates/etc/rc.d/init.d/masq/10masq_ftp
--- e-smith-packetfilter-1.16.0/root/etc/e-smith/templates/etc/rc.d/init.d/masq/10masq_ftp	1969-12-31 19:00:00.000000000 -0500
+++ mezzanine_patched_e-smith-packetfilter-1.16.0/root/etc/e-smith/templates/etc/rc.d/init.d/masq/10masq_ftp	2006-07-18 14:38:49.000000000 -0400
@@ -0,0 +1,2 @@
+/sbin/modprobe ip_nat_ftp
+/sbin/modprobe ip_conntrack_ftp
diff -Nur -x '*.orig' -x '*.rej' e-smith-packetfilter-1.16.0/root/etc/e-smith/templates/etc/rc.d/init.d/masq/40masqLAN mezzanine_patched_e-smith-packetfilter-1.16.0/root/etc/e-smith/templates/etc/rc.d/init.d/masq/40masqLAN
--- e-smith-packetfilter-1.16.0/root/etc/e-smith/templates/etc/rc.d/init.d/masq/40masqLAN	1969-12-31 19:00:00.000000000 -0500
+++ mezzanine_patched_e-smith-packetfilter-1.16.0/root/etc/e-smith/templates/etc/rc.d/init.d/masq/40masqLAN	2006-07-18 14:38:49.000000000 -0400
@@ -0,0 +1,8 @@
+    /sbin/iptables --table nat --new-chain PostroutingOutbound
+    /sbin/iptables --table nat --append PostroutingOutbound \
+	--source $OUTERNET -j ACCEPT
+    /sbin/iptables --append PostroutingOutbound -t nat -j MASQUERADE
+    if [ -n "$OUTERIF" ]; then
+        /sbin/iptables --append POSTROUTING -t nat \
+	    --out-interface $OUTERIF -j PostroutingOutbound
+    fi
diff -Nur -x '*.orig' -x '*.rej' e-smith-packetfilter-1.16.0/root/etc/e-smith/templates/etc/rc.d/init.d/masq/90adjustMasq mezzanine_patched_e-smith-packetfilter-1.16.0/root/etc/e-smith/templates/etc/rc.d/init.d/masq/90adjustMasq
--- e-smith-packetfilter-1.16.0/root/etc/e-smith/templates/etc/rc.d/init.d/masq/90adjustMasq	1969-12-31 19:00:00.000000000 -0500
+++ mezzanine_patched_e-smith-packetfilter-1.16.0/root/etc/e-smith/templates/etc/rc.d/init.d/masq/90adjustMasq	2006-07-18 14:38:49.000000000 -0400
@@ -0,0 +1,2 @@
+    /sbin/iptables --table nat --replace PostroutingOutbound 1 \
+	--source $OUTERNET -j ACCEPT
